metricbeat/etc/beat.yml

```diff
@@ -1,25 +1,21 @@
 metricbeat:
-  modules:
-    #golang:
-    #  metrics:
-    #    expvar:
-    #      period: 10s
-    apache:
+  metricsets:
```

Comment:
@tsg Any good ideas for the naming here? It is somehow duplicated with line 4 :-( I was tempted to call it just "sets"